packages/extensions/ws/src/index.ts

```diff
@@ -298,14 +298,11 @@ class WebSocketExtension extends SdkExtension {
     if (Date.now() > this.wsTokenExpiresAt) {
       const r = await this.rc.post('/restapi/oauth/wstoken');
       this.wsToken = r.data as WsToken;
-      this.wsTokenExpiresAt = Date.now() + (this.wsToken.expires_in - 10) * 1000;
+      this.wsTokenExpiresAt = Date.now() + (this.wsToken.expires_in * 0.8) * 1000;
```

Comment:
I'd suggest leaving a comment saying that the default expire time is 10 min, and that's why the 0.8 makes sense.
If the default expire time changes in the future, we can come back and revise this.

Reply:
Done